modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json

```diff
@@ -2080,7 +2080,7 @@
                   "type": {
                     "type": "string",
                     "example": "compensation",
-                    "description": "Type of Intent To File being submitted.",
+                    "description": "Type of Intent To File being submitted. For type \"survivor\", the request must include claimant_ssn or participant_claimant_id and be made by a valid Veteran Representative. If the Representative is not a Veteran or a VA employee, this method is currently not available to them, and they should use the Benefits Intake API as an alternative.",
```

Comment:
shouldn't these names be camelCase instead of snake_case to align with documentation?